builds/e2e/templates/nested-get-secrets.yaml

```diff
@@ -4,7 +4,7 @@ steps:
     inputs:
       azureSubscription: $(az.subscription)
       KeyVaultName: $(azure.keyVault)
-      SecretsFilter: 'edgebuilds-azurecr-io-username,edgebuilds-azurecr-io-pwd,kvLogAnalyticWorkspaceId,kvLogAnalyticSharedKey,EdgeConnectivityStorageAccountConnString'
+      SecretsFilter: 'edgebuilds-azurecr-io-username,edgebuilds-azurecr-io-pwd,kvLogAnalyticWorkspaceId,kvLogAnalyticSharedKey,EdgeConnectivityStorageAccountConnString,GitHubAccessToken'
```

Comment:
why do we need `GitHubAccessToken` ?

Reply:
Not used for us. Used y new IIOT to get library